pmultiqc/modules/common/file_utils.py

```
@@ -124,6 +125,11 @@ def extract_tar(file_path: str, extract_to: str) -> None:
         log.info(f"Extracted {file_path} to {extract_to}")
 
 
+def extract_rar(file_path, root_dir):
+    with rarfile.RarFile(file_path) as rf:
+        rf.extractall(root_dir)
```

Ruff (0.14.13)

Uses of `tarfile.extractall()`

(S202)